dora/core/common/src/main/java/alluxio/util/OSUtils.java

```diff
@@ -28,6 +28,9 @@ public final class OSUtils {
   public static final String JAVA_VENDOR_NAME = System.getProperty("java.vendor");
   /** Indicates the current java vendor is IBM java or not. */
   public static final boolean IBM_JAVA = JAVA_VENDOR_NAME.contains("IBM");
+  /** The maximum path length supported by a domain socket on unix varies depending on the
+   * operating system. The conservative limit is returned here */
+  public static final int UNIX_SOCKET_MAX_PATH_LENGTH = 100;
```

Comment:
Isn't it 108, why set it to 100?

Reply:
There are differences in the maximum length of domain socket paths among different operating systems. Therefore, it is recommended to limit the length of the socket path to the smallest common length range, such as around 100 characters.